picard/ui/itemviews.py

```diff
@@ -467,9 +475,16 @@ def drop_urls(urls, target):
         files = []
         new_files = []
         for url in urls:
+            log.debug("Dropped the URL: %r", url.toString(QtCore.QUrl.RemoveUserInfo))
             if url.scheme() == "file" or not url.scheme():
-                # Dropping a file from iTunes gives a filename with a NULL terminator
-                filename = os.path.normpath(os.path.realpath(unicode(url.toLocalFile()).rstrip("\0")))
+                # Workaround for https://bugreports.qt.io/browse/QTBUG-40449
+                # OSX Urls follow the NSURL scheme and need to be converted
+                if unicode(url.path()).startswith('/.file/id=') and NSURL_IMPORTED:
```

Comment:
Move test for `NSURL_IMPORTED` in first position in the `and` expression (faster on other platforms)